.changeset/tasty-dragons-train.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudoperators/juno-ui-components": minor
+---
+
+Marked AppIntro as deprecated in Storybook
```

Comment:
A lot of the changes here are internal. Does it make sense to have the below `changeset` message? As it's more accurate if anything went wrong...

```suggestion
Marked AppIntro as deprecated in Storybook and moved deprecated components to the deprecated_js folder in the code base (internal change)
```

Or just mention the external change?